src/quacc/recipes/espresso/phonons.py

```diff
@@ -463,3 +480,135 @@ def _grid_phonon_subflow(
+    parallel_info: dict[str] | None = None,
+    **calc_kwargs,
+) -> RunSchema:
+    """
+    Function to carry out a basic dvscf_q2r calculation. It should allow you to
+    use all the features of the dvscf_q2r binary which does not have an official
+    documentation.
+
+    The potential perturbation from a single atom's displacement is spatially localized,
+    allowing phonon potential interpolation from coarse to fine q-point grids using Fourier
+    interpolation. To use this, run ph.x on a coarse q-point grid, use dvscf_q2r.x to
+    inverse Fourier transform the phonon potentials to a real-space supercell, and then set
+    ldvscf_interpolation=.true. in ph.x to Fourier transform the potentials to desired q points.
+
+    Only one card, &input:
+
+    prefix  : Prepended to input/output filenames, default: 'pwscf'
+    outdir  : Directory containing input, output, and scratch files.
+              In quacc this is always set to the current working directory.
+    fildyn  : File where the dynamical matrix is written.
+              In quacc this should always be set to 'matdyn'.
+    fildvscf : File where the potential variation is written.
+               In quacc this should always be set to 'dvscf'.
+               (character, Default: 'dvscf')
+    wpot_dir : Directory where the w_pot binary files are written.
+               In quacc this is always set to outdir / w_pot
+    do_long_range : If .true., subtract the long-range part of the potential
+                    before interpolation. Requires epsilon and Born effective
+                    charge data in _ph0/prefix.phsave/tensor.xml. default: .false.
+    do_charge_neutral : If .true., renormalize phonon potential to impose
+                    neutrality of Born effective charges. default: .false.
+    verbosity : If 'high', write more information to stdout.
+
+    Parameters
+    ----------
+    copy_files
+        Source directory or directories to copy files from. If a `SourceDirectory` or a
+        list of `SourceDirectory` is provided, this interface will automatically guess
+        which files have to be copied over by looking at the binary and `input_data`.
+        If a dict is provided, the mode is manual, keys are source directories and values
+        are relative path to files or directories to copy. Glob patterns are supported.
+    parallel_info
+        Dictionary containing information about the parallelization of the
+        calculation. See the ASE documentation for more information.
+    **calc_kwargs
+        Additional keyword arguments to pass to the Espresso calculator. Set a value to
+        `quacc.Remove` to remove a pre-existing key entirely. See the docstring of
+        [quacc.calculators.espresso.espresso.Espresso][] for more information.
+
+    Returns
+    -------
+    RunSchema
+        Dictionary of results from [quacc.schemas.ase.summarize_run][].
+        See the type-hint for the data structure.
+    """
+
+    calc_defaults = {}
```

Comment:
You can just set `calc_defaults={}` in the `run_and_summarize()` function here and elsewhere IMO.